lang/julia/Portfile

```diff
@@ -22,124 +22,82 @@ long_description    Julia is a high-level, high-performance dynamic programming
 
 github.tarball_from releases
 
-distfiles           ${name}-${version}-full${extract.suffix}
+distfiles           ${name}-${version}-full${extract.suffix} \
+                    ${name}-${version}-full${extract.suffix}.asc
 
-checksums           rmd160  c5a50b478cbef04ce2937c46173ebc63d9403934 \
-                    sha256  1e34c13091c9ddb47cf87a51566d94a06613f3db3c483b8f63b276e416dd621b
+checksums           ${name}-${version}-full${extract.suffix} \
+                    rmd160  8eac1e29a71bac430c93810afac15be07c8ed62b \
+                    sha256  3c5395dd3419ebb82d57bcc49dc729df3b225b9094e74376f8c649ee35ed79c2 \
+                    size    96079453 \
+                    ${name}-${version}-full${extract.suffix}.asc \
+                    rmd160  5b43a22ac7c852639fb251cc94a16c012efd7629 \
+                    sha256  4553930eae7745123dc6229557c8ace54b1a1e4f4f1d979e87d3578593581546 \
+                    size    866
 
-# Force buildbot to skip this since julia builds its own blas; see https://trac.macports.org/ticket/45187
+# Force buildbot to skip this since julia builds its own blas; see #45101
```

Comment:
Is that to be a trac ticket or a github ticket? If it's trac, the full URL is useful.

Reply:
That's in the original Portfile. Not sure. I'm inclined to leave it as is in case anyone ever needs to track it down.

Reply:
That directive and comment are both gone now.